src/components/overview/teamIntervalsOverview.tsx

```diff
@@ -0,0 +1,178 @@
+import {
+  XAxis,
+  Tooltip,
+  TooltipProps,
+  ResponsiveContainer,
+  Line,
+  LineChart,
+  AreaChart,
+  Area,
+} from "recharts";
+import {
+  ValueType,
+  NameType,
+} from "recharts/types/component/DefaultTooltipContent";
+import * as React from "react";
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { TeamIntervalsOverviewDto } from "@/app/api/teams/overview/intervals/route";
+
+const CustomTooltip = ({
+  active,
+  payload,
+}: TooltipProps<ValueType, NameType>) => {
+  if (active && payload && payload.length) {
+    return (
+      <div className="custom-tooltip">
+        {payload.map((data) => {
+          return (
+            <div>
+              <p className="label">{`${data.dataKey}: ${parseFloat(data.value!.toString()).toPrecision(3)}`}</p>
+            </div>
+          );
+        })}
+      </div>
+    );
+  }
```

Comment:
Move to a component?

Reply:
I think this one is too specific, the only reusable part is the class definition, specially now that I updated every chart for dark mode, the returns are completely different